gdpr/vendorlist-fetching_test.go

```diff
@@ -284,12 +292,6 @@ var vendorList2Expected = testExpected{
 	vendorPurposes:    map[int]bool{1: false, 2: true, 3: true},
 }
 
-var vendorListFallbackExpected = testExpected{
```

Comment:
@zhongshixi this code block is not used at all so i deleted it

Reply:
Did you need to delete it to accomplish your goal in this PR ?

Reply:
but why we need to keep it if the code is not used at all?   there is no comment or description associated with the deleted test case